tests/end_to_end/README.md

```diff
@@ -1,21 +0,0 @@
-# End-to-end Test
```

Comment:
I suggest to keep the readme. just stating that we by default run an e2e test in the bed and show how to find it and stop in

Reply:
When deployed by kubernetes, admin could easily stop it by dashboard or kubectl.

Reply:
added in `service.yaml`